google-cloud-storage/src/main/java/com/google/cloud/storage/StorageOptions.java

```diff
@@ -117,7 +117,7 @@ protected boolean projectIdRequired() {
     return SCOPES;
   }
 
-  protected StorageRpc getStorageRpcV1() {
+  public StorageRpc getStorageRpcV1() {
```

Comment:
I would prefer to keep this `protected` and cast it in the notification code, in order to prevent external dependencies on it.